tests/udp.rs

```diff
@@ -149,8 +171,8 @@ mod udp_tracker_server {
     }
 
     /// Creates a new UdpTrackerClient connected to a Udp Tracker server
-    async fn new_connected_udp_tracker_client(remote_address: &str) -> UdpTrackerClient {
-        let udp_client = new_connected_udp_client(remote_address).await;
+    async fn new_connected_udp_tracker_client(remote_socket: &SocketAddr) -> UdpTrackerClient {
```

Comment:
@da2ce7 I like this change. We should try to normalize this variable name. We are using:

- remote_address
- remote_socket
- remote_ip
- remote_client
- client?

I think `client_socket` is better because it's more specific. But I have not changed to avoid introducing more versions. 

And I've just realized in this case, it's the `server_socket` which makes my assumptions more valid that we should be more specific.